app/src/main/kotlin/compiler/intermediate_form/DataFlowAnalyser.kt

```diff
@@ -0,0 +1,110 @@
+package compiler.intermediate_form
+
+import compiler.common.reference_collections.ReferenceMap
+import compiler.common.reference_collections.ReferenceSet
+import compiler.common.reference_collections.referenceHashMapOf
+import compiler.common.reference_collections.referenceHashSetOf
+import compiler.common.reference_collections.referenceMapOf
+
+abstract class DataFlowAnalyser<SL> {
```

Comment:
```suggestion
abstract class DataFlowAnalyzer<SL> {
```
let's be consistent ;)